### ISSUE TITLE
Low Accuracy on ilsvrc12_val using VGG16 pre-trained network.

### ISSUE THREAD

**Issue by maxkohlbrenner** [keras-team/keras-applications#99]
Hi,
according to https://keras.io/applications/#vgg16, the vgg16 pre-trained model has a validation top-1 accuracy of 71.3%. 
I deployed the network with the given weights and preprocessing function and arrived at a top-1 accuracy of 64.2% on the ilsvrc12 validation set. 
Can anyone confirm the reported accuracy of the pre-trained network?

Thank you very much for your help!
Cheers,
Max

**Comment by taehoonlee** [comment]
@maxkohlbrenner,

I will share my inference codes in the near future. Key recipes for ImageNet are the following:
- Down-sampling should not be performed as a square.
Adjust shorten side as 256 is the right way.
In other words, a resized image is either one of (256 X W) or (H X 256).
- The down-sampling scheme is pretty important.
I evaluated all the models with `cv2.resize(img, (h, w), interpolation=cv2.INTER_CUBIC)`,
while the keras default is `img.resize((w, h), pil_image.NEAREST)`.
You can try `pil_image.BICUBIC` or opencv2 like me.
- After the down-sampling, you should crop the 224x224 center region from an (256 X W) or (H X 256) image.

**Comment by maxkohlbrenner** [comment]
Thank you very much  @taehoonlee!
This solved my problem.
